src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((item) => {
+    const child = item.querySelector('ul');
+
+    if (!child) {
+      return;
+    }
+
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent.trim();
+    item.firstChild.replaceWith(span);
+
+    child.style.display = 'block';
```

Comment:
It's not necessary to set the display property to 'block' here, as it's the default value for 'ul' elements.